wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1851,3 +1853,37 @@ class ApproximateConstantViolation(ASTViolation):
 
     code = 447
     error_template = 'Found approximate constant: {0}'
+
+
+@final
+class AlmostSwappedViolation(ASTViolation):
+    """
+    Forbids unpythonic swap variables.
+
+    We check for a = b; b = a sequences.
```

Comment:
We use double backquotes to highlight the code. Please, add it to `a = b; b = a`

Reply:
done